maquetado/login.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Login</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class='login-container'>
+      <div class='login-bottom'>
```

Comment:
This div is not necessary, you can move the padding attributes that have this class to `login-container`

Reply:
Done!